applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -406,6 +406,104 @@ static bool sink_parse_cb(struct bt_data *data, void *user_data)
 	return true;
 }
 
+static void set_color_if_supported(char *str, uint16_t bitfield, uint16_t mask)
+{
+	if (bitfield & mask) {
+		strcat(str, COLOR_GREEN);
+	} else {
+		strcat(str, COLOR_RED);
+	}
+}
+
+static bool cap_print_cb(struct bt_data *data, void *user_data)
+{
+	if (data->type == BT_AUDIO_CODEC_CAP_TYPE_FREQ) {
+		uint16_t freq_bit = sys_get_le16(data->data);
+		char supported_freq[320] = "";
```

Comment:
320 looks too large.

Reply:
Discussed offline, and they need to be this size because the color takes up 10 bytes. This will be linked away if `CONFIG_BT_AUDIO_EP_PRINT=n`